CONTRIBUTING.rst

```diff
@@ -1,53 +1,47 @@
 Contributing Guidelines
 =======================
 
-To learn more about the purpose of PEPs and how to go about writing a PEP, please
-start reading at PEP 1 (`pep-0001.txt <./pep-0001.txt>`_ in this repo). Note that
-PEP 0, the index PEP, is now automatically generated, and not committed to the repo.
+To learn more about the purpose of PEPs and how to go about writing one, please
+start reading at `PEP 1 <https://www.python.org/dev/peps/pep-0001/>`_.
```

Comment:
Does the github renderer support `` :pep`1` ``?

Reply:
I thought it didn't when I tested it a few looks ago, which is why I didn't use it here, but it looks like that's only for linking subsections and/or custom link text. So, we _could_ use it in this specific case, but a standard link would still need to be used for the specific-section reference below, and it also won't be too helpful for anyone viewing the Readme or Contributing Guide locally. So, IMO, I suggest just consistently using standard links in the readme and contributing guide here.